dts/common/nordic/nrf54l09.dtsi

```diff
@@ -33,15 +33,17 @@
 			itm: itm@e0000000 {
 				compatible = "arm,armv8m-itm";
 				reg = <0xe0000000 0x1000>;
-				swo-ref-frequency = <DT_FREQ_M(128)>;
+				/* NRFX-6881: Restore to 128MHz when PDK becomes available. */
+				swo-ref-frequency = <DT_FREQ_M(64)>;
```

Comment:
no sure what to do about it, If it's the same frequency as CPU then it should be derived from the parent. It's used here: https://github.com/zephyrproject-rtos/zephyr/blob/5bc6446dcc317d3f3cff49b703ad32e8ecb724cd/subsys/logging/backends/Kconfig.swo#L15 but probably Kconfig DT function does not support getting property from phandle.

Anyway imo it would be better to move it to PDK like CPU freq since it is a limitation of pdk and not soc.